keyboards/yampad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,142 @@
+#include QMK_KEYBOARD_H
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layers {
+  _BL = 0,
+  _NV,
+  _FN
+};
+
+enum custom_keycodes {
+  KC_DBL0 = SAFE_RANGE,
+};
```

Comment:
Just some formatting nitpicks
```suggestion
enum layers {
    _BL,
    _NV,
    _FN
};

enum custom_keycodes {
    KC_DBL0 = SAFE_RANGE,
};
```